src/gc/gcpriv.h

```diff
@@ -781,6 +779,9 @@ class generation
     int             gen_num;
 
 #ifdef FREE_USAGE_STATS
+    #pragma message ("FREE_USAGE_STATS changes the layout of the 'generation' class, which is known to the DAC. " \
```

Comment:
Unfortunately `dac_generation` is the one type in `gcinterface.dac.h` with which I need to do pointer arithmetic. Jan suggested above that make the size a dacvar anyway and do the pointer math myself in the DAC - it would give some additional flexibility in this case. Do you or @Maoni0 have any thoughts? (I'm not familiar with what this feature define is used for in the GC)